Rewrite installation doc for more detailed guide (#958)
oam-dev/kubevela.io: docs/install.mdx

---

> title: Guide

Reviewer: Overview of Installation

---

> Here are available installation options depends on your environment:

Reviewer: ```suggestion
Here are two installation methods available according to your environment:
```

### Don't have Kubernetes cluster yet?

> For those who are new to KubeVela or Kubernetes, installing KubeVela in standalone mode is the best choice.

Reviewer: ```suggestion
For those who are new to KubeVela and Kubernetes, installing KubeVela in standalone mode is the best choice. It doesn't require you to have any Kubernetes knowledge to experience KubeVela features.
```

> We developed [VelaD](https://github.com/kubevela/velad) for this purpose. VelaD package Kubernetes and KubeVela requirements to a single binary.

Reviewer: ```suggestion
We developed [VelaD](https://github.com/kubevela/velad) for this purpose, that we packages KubeVela along its dependencies including Kubernetes into a single binary.
```

Please follow the document linked below to install KubeVela.

> - [Standalone Installation](./installation/standalone): Install Kubernetes and KubeVela on local machine or a remote Linux server which you can access its IP address (Public Cloud or On-premise).

Reviewer: ```suggestion
- [Standalone Installation](./installation/standalone): Install KubeVela on a local machine or a remote server(Public Cloud or On-premise) based on Linux, MacOS or Windows operating system.
```

:::info

> VelaD suits great for local development and quick demos, while we strongly recommend you to [install KubeVela with managed Kubernetes services](./installation/kubernetes) for production usage.

Reviewer: ```suggestion
VelaD suits great for local development and quick demos, it leverages [K3s](https://github.com/k3s-io/k3s) to manage Kubernetes. We strongly recommend you to [install KubeVela with managed Kubernetes services](./installation/kubernetes) for production usage.
```
